data/Nano_2011/metadata.yml

```diff
@@ -0,0 +1,187 @@
+    we included 15 dune crest sites that were characterised by medium-stature shrubs
+    and sparse ground cover. The dune crest sites were included in the analysis to
+    test for the influence of fine-scale fire refugia in hummock grassland dune fields
+    given that grass fuel loads are usually low on crests relative to those on adjacent
+    dune slopes. The sites were from seven southern NT bioregions that are characterised
+    by these habitat types and that are periodically burned by wildfire (Fig. 1).
+    Most of the included sites (42%) were from the MacDonnell Ranges Bioregion, reflecting
+    an existing sampling bias that could not be avoided in this study. All available
+    data for sites not associated with the MacDonnell Ranges were used. The selection
+    of sites from the MacDonnell Ranges data set (c. 70% of relevant sites) aimed
+    to reflect habitat variability and the spatial extent of that system. For both
+    grassland and shrubland/woodland habitat, 22% of sites were from areas with a
+    yearly average rainfall of <250 mm and the remaining were from areas with between
+    250 and 450 mm. Sites were evenly distributed between course-scale runoff (mountain
+    range) and runon (plain/floodplain/dune field) settings. Sites were included only
+    if they had >10% cover of the structural dominant (excluding the crest sites)
+    and they had not been burned within 5 years of the survey. If a site had >10%
+    cover of any of the focal Acacia shrubland/woodland dominants, it was allocated
+    to the shrubland/woodland class, independent of the cover of Triodia in the understorey.
+    Of the grassland sites, 96 were dominated by resprouting Triodia species and 89
+    by non-resprouting Triodia species. We excluded Triodia grassland sites on chemically/texturally
+    extreme soils (sensu Nano and Clarke 2008) that are unlikely to support shrubland
+    regardless of fire effects. For shrubland/woodland habitat, we included 140 sites
+    dominated by fire-killed species (Acacia aneura, A. macdonelliensis and A. ammobia)
+    and 50 dominated by resprouting species (A. kempeana and A. estrophiolata). Site
+    allocation approximately reflected the availability of habitat samples and their
+    relative abundance in the landscape (Fig. 1). ^p Persistence traits ^p Persistence
+    trait data were collated for all of the woody species that were recorded from
+    the 385 sites. Population-level data were derived from published (e.g. Crowley
+    et al. 2007) and unpublished databases (e.g. NT Ecological Attributes Database,
+    Northern Territory Government), the scientific literature and our own field records.
+  original_file: Data copied from table from manuscript
+  notes: .na
+locations:
+  Southern region of the Northern Territory:
+    latitude range (deg): -19 to -26
+    longitude range (deg): 129 to 138
+    latitude (deg): .na
```

Comment:
This is the right solution for now, but in the long run, it would be good to have a way to input regional lat/lon.